.github/workflows/code-analysis.yml

```
@@ -1,83 +1,83 @@
-          Write-Information -MessageData 'Converting PSScriptAnalyzer result to SARIF.' -InformationAction 'Continue'
-          $pssaError |
-            ConvertTo-SARIF -FilePath 'results.sarif'
-
-          Write-Information -MessageData 'Analyzing done.' -InformationAction 'Continue'
-      - name: Upload SARIF results
-        uses: github/codeql-action/upload-sarif@v3
-        with:
-          sarif_file: results.sarif
+name: Code analysis (source)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)